doc/specs/#166 - Font Support.md

```diff
@@ -0,0 +1,416 @@
+| Narrow Italic | 7.01 | Arial | C:\Windows\Fonts\ARIALNI.TTF |
+| Narrow Bold | 7.01 | Arial | C:\Windows\Fonts\ARIALNB.TTF |
+| Narrow Bold Oblique | 7.01 | Arial | C:\Windows\Fonts\ARIALNB.TTF |
+
+> Note that some font faces share the same font file path. This will need to be handled properly during uninstall.
+
+Sample Code:
+
+```c++
+        // Obtaining all font file paths associated with a font face.
+        wil::com_ptr<IDWriteFontFace> fontFace;
+        THROW_IF_FAILED(font->CreateFontFace(fontFace.addressof()));
+
+        // https://learn.microsoft.com/en-us/windows/win32/api/dwrite/nf-dwrite-idwritefontface-getfiles
+        UINT32 fileCount;
+        THROW_IF_FAILED(fontFace->GetFiles(&fileCount, nullptr)); // Use null initially to retrieve the number of files.
+
+        wil::com_ptr<IDWriteFontFile> fontFiles[8];
+        THROW_HR_IF(E_OUTOFMEMORY, fileCount > ARRAYSIZE(fontFiles));
+        THROW_IF_FAILED(fontFace->GetFiles(&fileCount, fontFiles[0].addressof()));
+
+        // Iterate through all the files corresponding to that font face name.
+        for (UINT32 i = 0; i < fileCount; ++i) {
+            wil::com_ptr<IDWriteFontFileLoader> loader;
+            THROW_IF_FAILED(fontFiles[i]->GetLoader(loader.addressof()));
+
+            const void* fontFileReferenceKey;
+            UINT32 fontFileReferenceKeySize;
+            THROW_IF_FAILED(fontFiles[i]->GetReferenceKey(&fontFileReferenceKey, &fontFileReferenceKeySize));
+
+            // Retrieve and print the full font file path.
+            if (const auto localLoader = loader.try_query<IDWriteLocalFontFileLoader>()) {
+                UINT32 pathLength;
+                THROW_IF_FAILED(localLoader->GetFilePathLengthFromKey(fontFileReferenceKey, fontFileReferenceKeySize, &pathLength));
+                pathLength += 1; // Account for the trailing null terminator during allocation.
+
+                wchar_t path[512];
+                THROW_HR_IF(E_OUTOFMEMORY, pathLength > ARRAYSIZE(path));
+                THROW_IF_FAILED(localLoader->GetFilePathFromKey(fontFileReferenceKey, fontFileReferenceKeySize, &path[0], pathLength));
+
+                // Normalize path for better readability
+                // Use backup semantics so we can access protected files.
+                const auto h = CreateFileW(path, 0, 0, nullptr, OPEN_EXISTING, FILE_FLAG_BACKUP_SEMANTICS, nullptr);
+                GetFinalPathNameByHandleW(h, path, ARRAYSIZE(path), 0);
+                CloseHandle(h);
+
+                wprintf(L"%s\n", &path[0]); 
+            }
+        }
+```
+
+`winget fonts install`
+---
+
+### 1. Selection From Font Source
+Font source:
+There will be a separate source specifically for fonts. This source will be marked as explicit. The workflows following font source selection will be the same as if an executable package was selected. A user will be enter the following commands and experience the same behavior of installing a font.
```

Comment:
How will this new source interact with the GPO for allowed / disallowed sources?

Reply:
Good question for @denelon 

The new source was just my thought on how we would separate the data and keep the index size from growing.  My expectation is that it would respect configured policy just the same, but the question is whether it would be in the umbrella of some existing controls or not (the one that controls the winget source for example).